scripts/goby_launch

```diff
@@ -193,56 +170,88 @@ gobyd_interprocess=$(bash -c "$gobyd_escline --app 'debug_cfg: true'" | sed -n "
 gnterm=()
 while IFS= read -r line
 do
-    escline=$(echo $line | sed 's/"/\\"/g')
-    bin=$(echo $line | cut -d " " -f 1)
+    cmd=$(echo $line | sed 's/^ *\[.*\]//')
+    directive_str=
+    echo $line | egrep -q '\[.*\]'  &&
+        directive_str=$(echo $line | sed 's/^ *\[\(.*\)\].*$/\1/')
```

Comment:
My suggestion is to use `\s` (if `sed` supports it) to allow for other whitespace in the indentation. The `.*$` is redundant.

Reply:
Actually, `.*$` isn't redundant if you want to clear the rest of the line, sorry.

Reply:
But you should quote `"$line"` when you're echoing it.